content/docs/pipelines/tutorials/pipelines-tutorial.md

```diff
@@ -72,28 +71,20 @@ Follow these steps to set up your GCP environment:
 1. Select or create a project on the [GCP Console][gcp-console].
 1. Make sure that billing is enabled for your project. See the guide to
   [modifying a project's billing settings][billing-guide].
-1. Install the [Cloud SDK][cloud-sdk].
+1. Grant users access to kubeflow with the `Owner` and `IAP-secured Web App User` roles:
+
+      ```
+      gcloud projects add-iam-policy-binding <PROJECT>
```

Comment:
Please remove this script block. I was wrong, project owner cannot be granted from the command line. They need to use Cloud Console.

Reply:
I adjusted this, removing the 'Owner' part but leaving the IAP role. Or do they all need to be removed?